test/metrics.test.js

```diff
@@ -41,6 +42,85 @@ describe("metrics", () => {
         });
     });
 
+    describe("container memory size", () => {
+        beforeEach( () => {
+            mockFs();
+        });
+
+        afterEach(() => {
+            mockFs.restore();
+        });
+
+        it("should return container memory size", () => {
+            mockFs({
+                '/sys/fs/cgroup': {
+                    'memory': {
+                        'memory.limit_in_bytes': '9999'
+                    }
+                }
+            });
+            const containerMemorySize = Metrics.containerMemory().containerMemorySize;
+            assert.ok(typeof containerMemorySize === "number");
```

Comment:
Ecxtreme nitpick: JS has also `Infinity` that technically is a (number too)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity]. The test should probably also check it's not a `+/- Infinity`, using `isFinite`. You can end up with an `Infinity` in error cases (e.g. divide by zero, depending on the implementations).

Reply:
but the next line checks that the number is as expected:
```
assert.equal(containerMemorySize, 9999);
```
so it will fail in the next line if it is +/- infinity

Reply:
but good to know! I also noticed this interesting behavior:
```
typeof NaN === 'number'
```

Reply:
Was more an FYI than anything else.

Reply:
For NaN there is also a special function, isNaN() to use.